gmmp/settings.py

```diff
@@ -287,3 +288,11 @@
     },
 ]
 
+# GSheets
+GSHEETS = {
+    'CLIENT_SECRETS': os.path.join(BASE_DIR, env.get("GSHEETS_CLIENT_SECRETS", ".env/credentials.json")),
+}
+GSHEETS_SPECIAL_QUESTIONS = {
+    'SPREADSHEET_ID': '1dttFvgcmS4yYrioge2Awav5k65QSAlSd95bX9yCqjA0',
```

Comment:
This is okay since the GMMP is a one-time thing but I would recommend putting the `SPREADSHEET_ID` into an env variable.

Reply:
Cool... I've made them ENV configurable just in case @esirK